_posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Downsampling
+nav: Downsampling
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions downsampling
+---
+
+Downsampling data allows to reduce the amount of data by simplifying the old values.
+
+## Example
+
+Bellow an example of a logic of downsampling. It is based on TimescaleDB documentation.
+
+The data is stored in a table `conditions`, composed by `temperature` and `humidity`
+value. This is a use case where a table is composed of raw datas.
```

Comment:
data is still uncountable. Do you have a dictionary installed in the text editor you use to write doc? It should highlight that `datas` does not exist

```suggestion
value. This is a use case where a table is composed of raw data.
```